### ISSUE TITLE
Implement token for accessible border width

### ISSUE THREAD

**Issue by lovemecomputer** [carbon-design-system/carbon#2437]
on tabs/etc that use a border to highlight the current location, the border should be a minimum of `3px` to meet accessibility standards. this is a a thing i've had to implement in a couple places on the website. 

i recommend making a new token for this width, and then doing a pass thru all of our components to make sure any 2px borders are bumped up to this

**Comment by shixiedesign** [comment]
Hey @lovemecomputer , how are you planing to add the token for border? is it possible to make it scalable to include more border tokens? Eg. we need borders applied to buttons now (dark themes only), and these will be ideally tokenized so themeing could remove / customize it.

**Comment by lovemecomputer** [comment]
@shixiedesign  sure, we could add any tokens that make sense to be a part of the system! it would probably be best to open a separate Issue for that, especially if you have in mind particular components the tokens apply to. that could potentially fit inside of components, since it's border measurements specifically for buttons.

**Comment by lovemecomputer** [comment]
thoughts on naming/organization of this token:

i asked @aagonzales over slack about filing this under a `layout` or `spacing` name. anna said:

> I dont think those, the `spacing` and `layout` tokens are for the empty spaces. You wouldn't use it for a border. Spacing essentially equals `margin` or `padding`.

> I think you thinking more of something that would be like `sizing`. Which we could add but its kind of like the mini unit increments.

good note on a set of `sizing` tokens. 

@shixiedesign `sizing` token category might be good for the button borders you mentioned as well!
